### ISSUE TITLE
Tests and Tasks

### ISSUE THREAD

**Issue by phucdev** [phucdev/TL_Bio_RE#4]
### Problems
We observed a significant performance loss. Possible causes:

- Reduction of the batch size from 32 down to 8 
- Changing from pairwise to sentence wise train-dev-test splits (major cause)
- New weird label distribution (with sentence wise split)

### Analysis
Look at the data:
- [x] Look at label distribution: interaction=True vs. interaction=False, maybe explore different metrics or adapt the way we work with our training data
- [x] Check sentence lengths in the corpora (again).

- [x] Use Korpus-Bearbeiten.ipynb to recreate our original pair wise split on the corpus data.

Tests, that we can then do without much effort:
- [x] Test our original Colab notebook on pair wise splits. 
- [x] Test with different batch sizes. 

### Nexts steps

- [x] Clean up model and predictor code
- [x] Add implementation for #1
